kernel_gateway/tests/test_gatewayapp.py

```diff
@@ -524,6 +524,68 @@ def test_seed(self):
 
         ws.close()
 
+class TestSeedGatewayAppKernelLanguageSupport(TestGatewayAppBase):
+    def setup_app(self):
+        self.app.prespawn_count = 1
+        self.app.seed_uri = os.path.join(RESOURCES,
+                                         'zen{}.ipynb'.format(sys.version_info.major))
+    @coroutine
+    def spawn_kernel(self):
+        ws = yield super(TestSeedGatewayAppKernelLanguageSupport, self).spawn_kernel(kernel_body="{{\"name\":\"python{}\"}}".format(sys.version_info.major))
```

Comment:
Nit. Might just want to use json.dumps() on a dict for clarity instead of building a JSON string manually.